thirdeye/thirdeye-frontend/app/pods/self-service/manage/template.hbs

```diff
@@ -1,48 +1,64 @@
-<h1 class="te-title">Manage Alerts</h1>
-{{outlet}}
+<div class="te-title">Manage Alerts</div>
```

Comment:
I think `h1` might be more semantic. I know we might have to override a lot of bootstrap classes, perhaps we could do it in one mixin (@include reset-bootstrap-title) for all h* ?